src/index.ts

```diff
@@ -12,7 +12,7 @@ const PROPOSED_ANSWER_KEYWORD = core.getInput('proposed-answer-keyword', { requi
 const closeLockedDiscussionsInput = core.getInput('close-locked-discussions', { required: false });
 const CLOSE_LOCKED_DISCUSSIONS = closeLockedDiscussionsInput.toLowerCase() === 'true' ? true : false;
```

Comment:
This is all my bad, but either this will need to be adjusted the same way, or the documentation in the README should be updated to say that this is false by default. Leaving it up to you

Reply:
To err is human. :) I could change this variable CLOSE_LOCKED_DISCUSSIONS as well and make the corresponding changes in Readme for both variables.